doc/mk_sdcard_image.sh

```diff
@@ -286,18 +343,36 @@ unpack_domd()
 {
 	local db_base_folder=$1
 	local loop_dev=$2
-	local img_output_file=$3
 
 	print_step  "Unpacking DomD"
 
-	unpack_dom_from_tar $db_base_folder $loop_dev $img_output_file 2 domd
+	unpack_dom_from_tar $db_base_folder $loop_dev 2 domd
```

Comment:
we still use hard-coded "2" here, but should employ $DOMD_PARTITION?

Reply:
Fixed.